introduction.md

```diff
@@ -1,3 +1,9 @@
 Hi! I'm Nick, and this is my intro...
+
 My favorite path is the HTML & CSS path!
 
+Deadly SKills
+=============
+
+*Git
+*HTML & CSS
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?